code/game/objects/structures/watercloset.dm

```diff
@@ -141,6 +197,27 @@
 
 
 /obj/structure/urinal/attackby(obj/item/I, mob/user, params)
+	if(iswrench(I))
+		if(anchored)
+			user.visible_message("[user] begins disconnecting [src]...", "<span class='notice'>You begin to disconnect [src]...</span>")
+			if(do_after(user, 40 * I.toolspeed, target = src))
+				if(!loc || !anchored)
+					return
+				user.visible_message("[user] disconnected [src]!", "<span class='notice'>You disconnected [src]!</span>")
```

Comment:
Present continuous tense messages are generally preferred to past tense messages. So this should be `[user] disconnects [src]!` and `You disconnect [src]!`.